packages/esm-patient-search-app/src/compact-patient-search/recently-searched-patients.component.tsx

```diff
@@ -74,9 +74,16 @@ const RecentlySearchedPatients = React.forwardRef<HTMLDivElement, RecentPatientS
         <div className={styles.searchResultsContainer}>
           <div className={styles.searchResults}>
             <p className={styles.resultsText}>
-              {t('recentSearchResultsCount', '{{count}} recent search result', {
-                count: searchResults.length,
-              })}
+              <span className={styles.resultsTextCount}>
+                {t('recentSearchResultsCount', '{{count}} recent search result', {
+                  count: searchResults.length,
+                })}
+              </span>
+              {isValidating && (
+                <span className={styles.validationIcon}>
+                  <InlineLoading className={styles.spinner} />
+                </span>
+              )}
```

Comment:
This will render a loading spinner when we're revalidating recently searched patients. Additionally, because we've specified the [keepPreviousData](https://swr.vercel.app/docs/advanced/understanding.en-US#return-previous-data-for-better-ux) option in the `useInfinitePatientSearch` hook, the previous data will be returned while the new data is being fetched.